src/components/Gigs/Calendar.svelte

```diff
@@ -0,0 +1,435 @@
+      }
+    }
+  }
+  .gigtype-gig_cancelled {
+    @include themeifyThemeElement($themes) {
+      &.calendar-entry,
+      &.key-entry::before {
+        box-shadow: shadow(themed("formColor"));
+      }
+    }
+  }
+
+  .gigtype-calendar {
+    @include themeifyThemeElement($themes) {
+      &.calendar-entry,
+      &.key-entry::before {
+        box-shadow: shadow(themed("negative"));
+      }
+    }
+  }
+
+  .gigtype-gig.admins-only {
+    @include themeifyThemeElement($themes) {
+      &.calendar-entry,
+      &.key-entry::before {
+        box-shadow: shadow(themed("blueGig"));
+      }
+    }
+  }
+
+  .gigtype-kit {
+    @include themeifyThemeElement($themes) {
+      &.calendar-entry,
+      &.key-entry::before {
+        box-shadow: shadow(themed("kitHire"));
+      }
+    }
+  }
+
+  .today {
+    @include themeifyThemeElement($themes) {
+      font-weight: bolder;
+      color: themed("accent");
+      color: var(--accent);
+    }
+  }
+
+  .title-bar {
+    display: grid;
+    grid-template-columns: 1fr auto 1fr;
+    align-items: center;
+    justify-content: center;
+    margin: 0.5em 0;
+  }
+
+  .title-bar .left {
+    margin-right: auto;
+  }
+
+  .title-bar .right {
+    margin-left: auto;
+  }
+
+  .title-bar button {
+    padding: 0.1em 0.5em;
+    flex-basis: 200px;
+  }
+
+  .title-bar h3 {
+    flex-shrink: 0;
+    margin: 0;
+  }
+
+  .key.hidden {
+    max-height: 0;
+    opacity: 0;
+    transition: 0.3s all linear;
+    transition-delay: 0.1s;
+  }
+
+  .key {
+    max-height: calc(6 * 3em + 4em);
+    opacity: 1;
+    transition: 0.3s all linear;
+    overflow: hidden;
+    margin-bottom: 0em;
+  }
+
+  .key ul {
+    padding-left: 0.5em;
+    margin-top: 0;
+  }
+
+  .key.hidden li {
+    opacity: 0;
+  }
+
+  .key li {
+    opacity: 1;
+    transition: 0.3s all linear;
+    display: flex;
+    align-items: center;
+  }
+
+  .key li::before {
+    display: flex;
+    content: "24";
+    width: 2.5em;
+    height: 2em;
+    border-radius: 3px;
+    margin: 0.25em 0;
+    align-items: center;
+    justify-content: center;
+    margin-right: 0.5em;
+  }
+
+  .key-title {
+    margin: 0;
+  }
+
+  button.key-toggle {
+    margin: 0.5em auto;
+    display: block;
+  }
+
+  .key:not(.hidden) li {
+    transition-delay: var(--delay);
+  }
+
+  .key.hidden li {
+    transition-delay: 0;
+  }
+</style>
+
+<div class="title-bar">
+  <button on:click="{() => dispatchEvent('clickPrevious')}" class="left" data-test="gigcalendar-previous-month">
+    Prev
+  </button>
+  <h3>{displayedMonth.format('MMMM YYYY')}</h3>
+  <div class="right">
+    <button on:click="{() => (showSelection = !showSelection)}" title="Select month">
+      <i class="las la-calendar"></i>
+    </button>
+    <button on:click="{() => dispatchEvent('clickNext')}" data-test="gigcalendar-next-month">Next</button>
+  </div>
+</div>
+{#if showSelection}
+  <div class="month-selector">
+    <!-- svelte-ignore a11y-no-onchange -->
```

Comment:
? what does this do?

Reply:
Disables a warning about onchange on select, which is mostly redundant in modern browsers.